packages/eslint-config-vtex/rules/tests.js

```diff
@@ -0,0 +1,75 @@
+// Jest: https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules
+// Cypress: https://github.com/cypress-io/eslint-plugin-cypress/blob/master/docs/rules
+module.exports = {
+  overrides: [
+    // ! CYPRESS
+    {
+      files: ['**/cypress/**/*{.test,.spec,_spec}.{ts,tsx,js,jsx}'],
+      extends: ['plugin:cypress/recommended'],
+      rules: {
+        // Enforce assertions before taking a screenshot
+        // https://github.com/cypress-io/eslint-plugin-cypress/blob/master/docs/rules/assertion-before-screenshot.md
+        'cypress/assertion-before-screenshot': 'warn',
+      },
+    },
+    // ! JEST
+    {
+      // Run through every test file found
+      files: ['*.{test,spec}.{ts,tsx,js,jsx}'],
+      // Unless it's inside a cypress directory
+      excludedFiles: ['**/cypress/**'],
+      extends: ['plugin:jest/recommended', 'plugin:jest/style'],
+      rules: {
+        // Enforce consistent a test method name
+        // https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/consistent-test-it.md
+        'jest/consistent-test-it': [
```

Comment:
Nooo

Reply:
No what?? 🤔

Reply:
The consistent it rule? GITHUB, WHY DO YOU CONFUSE ME??

Reply:
I use both depending on the test file. If it's small, a `test` it's just fine.

Reply:
Done, ma booooy

Reply:
I can, however, make it use `test` for top-level tests and `it` for tests inside a `describe`

Reply:
> I can, however, make it use `test` for top-level tests and `it` for tests inside a `describe`

That's perfect